### PR TITLE
Add an api call for calculating postage rates

### PR DESCRIPTION
**Pull request by jarneson**

### CODE REVIEW

File: lib/endicia.rb [hunk 1]
```diff
@@ -373,6 +373,82 @@ def self.carrier_pickup_request(tracking_number, package_location, options = {})
...
+  #       :prices => Array of hashes containing price data
+  #     }
+  def self.calculate_postage_rates(options = {})
+    url = "#{label_service_url(options)}/CalculatePostageRatesXML"
+
+    mailpiece_dimensions = options.delete(:MailpieceDimensions)
+
+    services = options.delete(:Services)
+
+    xml = Builder::XmlMarkup.new.PostageRatesRequest do |xl|
```
Comment (kgleong) [inline lib/endicia.rb]: what is xl here?

**Comment by kgleong** [comment]
nice :+1: LGTM

File: lib/endicia.rb (hunk 1 above)
Comment (epixhints) [inline lib/endicia.rb]: the xml builder object, using `xml` masks the outer one, which isn't bad but gives lint errors

Comment (kgleong) [inline lib/endicia.rb]: yeah, figured that's what was going on.  could we find another name?  this seems like an ec2 instance size or something

Comment (kgleong) [inline lib/endicia.rb]: maybe just rename `xml` to `xml_body` so that `xl` can become `xml`.

**Pull request merged**